tools/topology/topology2/platform/intel/bt-generic.conf

```diff
@@ -198,7 +199,7 @@ Object.Base.route [
 	}
 	{
 		source	$BT_CP_DAI_PIPELINE_SRC
-		sink	$BT_CP_HOST_PIPELINE_SINK
+		sink	"host-copier.$BT_PCM_ID.capture"
```

Comment:
@plbossart I dont want to use the macro because for 1 if the PCM ID changes, the macro will also have to change. Its not reasy to keep track of the macro based on another macro. This way, the route sink is tied only to the PCM ID only not on the PCM ID and another macro that uses it.